### PR TITLE
Feature/api setup

### PR DESCRIPTION
**Pull request by cagallo**
#### What's this PR do?
- Add api calls with new server link
- Edit promise to make it functional on page load

#### Where should the reviewer start?

- apiCalls.js
- Scripts.js

#### How should this be manually tested?

- run npm start, go to localhost:8080

#### Any background context you want to provide?

N/A

#### What are the relevant tickets?

N/A

### CODE REVIEW

File: src/apiCalls.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
-let userAPI = fetch("https://what-s-cookin-starter-kit.herokuapp.com/api/v1/users")
+let userAPI = fetch("http://localhost:3001/api/v1/users")
```
Comment (anewb87) [inline src/apiCalls.js:1]: Ah. So nice. I love a good copy and paste.

**Review by kammon10 — APPROVED** [review]

File: src/scripts.js [hunk 2]
```diff
@@ -10,19 +14,22 @@ let userIndex;
...
 let ingredientsData;
+let recipeData;
 
 Promise.all([userAPI, ingredientAPI, recipeAPI])
   .then(data => {
-    ingredientsData = data[1].ingredientsData;
-    userIndex = getRandomIndex(data[0].usersData);
-    currentUser = new User(data[0].usersData[userIndex], data[1].ingredientsData);
-    recipeList = data[2].recipeData.map(recipe => {
+    [usersData, ingredientsData, recipeData] = [data[0], data[1], data[2]];
```
Comment (anewb87) [inline src/scripts.js:23]: Learned something new! Destructuring assignment for the win!

Comment (aliroemhildt) [inline src/scripts.js:23]: love how clean this syntax is!

**Review by anewb87 — APPROVED** [review]
Woot! This was a really smooth process. Everything appears to be working great.

**Review by aliroemhildt — APPROVED** [review]
APIs aren't as scary so scary after all!

**Pull request merged**